build/configs/common_download.sh

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env bash
```

Comment:
Could I request you to maintain consistency in terms of the tense of the verb "add"

> build/configs : Add common download script

vs.

> Added common_download.sh in build/configs

I think the latter should be adjusted to "Add"

Reply:
Done.